sorting/MERGENR.C

```diff
@@ -0,0 +1,82 @@
+/* Program to demonstrate non recursive merge sort */

```

Comment:
Add a sample test case here also

Reply:
This merge sort is implemented using a nonrecursive approach. Adding a sample test case.